functions/Set-DbaAgentJob.ps1

```diff
@@ -379,12 +379,12 @@ function Set-DbaAgentJob {
                 }
             }
 
-            if ($EventLogLevel) {
+            if ($null -ne $EventLogLevel) {
```

Comment:
This should use the `Test-Bound` internal function...

```suggestion
            if (Test-Bound -ParameterName EventLogLevel) {
```

Reply:
That's not the same though. Using `Test-Bound` is to know when the parameter has been explicitly specified when the function was called. What would happen if a function (for whatever reason) was called like this:
```
Set-DbaAgentJob ... -EventLogLevel $null
```
It's definitely gonna throw an error, as the value is missing. Which is why I decided to go with `$null -ne ...`

Reply:
If that is the case, there are 491 references in the module that should be updated 🎐

Reply:
I would recommend simply updating `Test-Bound` to validate for null as well. Our standard for validating whether a parameter was passed in as been to use this internal function (we just have not been enforcing it on all the older commands).

Reply:
I checked and in this particular scenario the value cannot be null, as enforced by ValidateSet. Will change to Test-Bound